packages/sankaku/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Context, Schema, trimSlash } from 'koishi'
+import { ImageSource } from 'koishi-plugin-booru'
+import { SankakuComplex } from './types'
+
+class SankakuComplexImageSource extends ImageSource<SankakuComplexImageSource.Config> {
+  languages = ['en']
+
+  constructor(ctx: Context, config: SankakuComplexImageSource.Config) {
+    super(ctx, config)
+  }
+
+  async get(query: ImageSource.Query): Promise<ImageSource.Result[]> {
+    // API docs: https://chan.sankakucomplex.com/cn/help/api
+    const params = {
+      tags: query.tags.join('+') + "+order:random",
+      limit: query.count
+    }
+    const url = trimSlash(this.config.endpoint) + 'posts?' + Object.entries(params).map(([key, value]) => `${key}=${value}`).join('&')
```

Comment:
```suggestion
    const url = trimSlash(this.config.endpoint) + 'posts.json?' + Object.entries(params).map(([key, value]) => `${key}=${value}`).join('&')
```

I think it would become json if you add `.json` extension without `Accept` header.

Reply:
> I think it would become json if you add `.json` extension without `Accept` header.

If I add `.json` here, it won't work. This API seems to be a new version without any documentation yet.